README.md

```diff
@@ -43,6 +43,8 @@ export ANSIBLE_STRATEGY="mitogen_linear"
 vagrant up
 # Configure TDP prerequisites
 ansible-playbook ansible_roles/collections/ansible_collections/tosit/tdp_prerequisites/playbooks/all.yml
+# Init "tdp_vars" with defaults from collections and customize them with overrides
+tdp init --overrides tdp_vars_overrides
```

Comment:
`tdp init` is already in the `setup` script and should not be here. Update the `setup` script.

Reply:
done